packages/modeling/src/measurements/measureBoundingBox.js

```diff
@@ -175,9 +175,21 @@ const measureBoundingBoxOfGeom3 = (geometry) => {
   return boundingBox
 }
 
+const fixBound = (i, v1,v2)=>{
```

Comment:
I don’t think that anyone will understand this function in one or two years. Please add a comment.

Reply:
:D np, let me know if the new comments are sufficient